Gems/LmbrCentral/Code/Tests/AxisAlignedBoxShapeTest.cpp

```diff
@@ -6,21 +6,23 @@
  *
  */
 
-#include <AzTest/AzTest.h>
-
 #include <AZTestShared/Math/MathTestHelpers.h>
 #include <AzCore/Component/ComponentApplication.h>
 #include <AzCore/Math/Matrix3x3.h>
 #include <AzCore/Math/Random.h>
 #include <AzCore/UnitTest/TestTypes.h>
 #include <AzFramework/Components/TransformComponent.h>
 #include <AzFramework/UnitTest/TestDebugDisplayRequests.h>
+#include <AzTest/AzTest.h>
 #include <Shape/AxisAlignedBoxShapeComponent.h>
+#include <ShapeTestUtils.h>
 #include <ShapeThreadsafeTest.h>
 
 namespace UnitTest
 {
-    class AxisAlignedBoxShapeTest : public AllocatorsFixture
+    class AxisAlignedBoxShapeTest
+        : public AllocatorsFixture
+        , public ShapeOffsetFixture
```

Comment:
question: Is inheriting from two fixtures a good idea? 🤔 Or is one not really a fixture, just checking there's no off multiple inheritance going on here 😝

Reply:
Renamed as suggested in your other comment.